bin/marked

```diff
@@ -78,7 +78,7 @@ function main(argv, callback) {
     arg = getarg();
     switch (arg) {
       case '--test':
-        return require('../test').main(process.argv.slice());
+        return require('../test').main(process.argv);
```

Comment:
`.slice()` is supposed to be there to pass a copy of the array to prevent unwanted changes directly to `process.argv`